test/unit/cpu/test-forall.cpp

```diff
@@ -104,3 +105,16 @@ using OpenMPTypes = ::testing::Types<
 
 INSTANTIATE_TYPED_TEST_CASE_P(OpenMP, ForallTest, OpenMPTypes);
 #endif
+
+#if defined(RAJA_ENABLE_TBB)
+using TBBTypes = ::testing::Types<
+    ExecPolicy<seq_segit, tbb_for_exec>,
+    ExecPolicy<tbb_for_exec, seq_exec>,
```

Comment:
I have no objection to that, but had removed it since it was just an alias.  Suppose it's best to just be consistent for now and we can just deprecate them all as a set later maybe.